node/metrics/src/runtime.rs

```diff
@@ -126,6 +128,23 @@ impl RuntimeMetricsProvider {
 			},
 		}
 	}
+
+	// Returns the `bs58` encoded metric update operation.
+	fn parse_event_params(event_params: &String) -> Vec<u8> {
+		println!("params: {}", event_params);
+
+		// Shave " }" suffix.
+		let new_len = event_params.len() - 2;
```

Comment:
The range calculations are all a bit too optimistic for my taste, and should assume too short or changed prefixes and hence use saturating arthmetic